docs/equality.md

```diff
@@ -7,28 +7,28 @@ We show the most common usages here, but if you need more ways to check for equa
 ## Primitive equality
 
 ```ts
-import { eq } from "fp-ts";
+import { boolean, date, number, string } from "fp-ts";
 
-eq.eqBoolean.equals(true, true); // true
-eq.eqDate.equals(new Date("1984-01-27"), new Date("1984-01-27")); // true
-eq.eqNumber.equals(3, 3); // true
-eq.eqString.equals("Cyndi", "Cyndi"); // true
+boolean.Eq.equals(true, true); // true
+date.Eq.equals(new Date("1984-01-27"), new Date("1984-01-27")); // true
+number.Eq.equals(3, 3); // true
+string.Eq.equals("Cyndi", "Cyndi"); // true
 ```
 
 ## Compare structures
 
 ```ts
 import { eq } from "fp-ts";
+import { Eq } from "fp-ts/lib/Eq";
 
 type Point = {
   x: number;
   y: number;
 };
 
-const eqPoint: eq.Eq<Point> = eq.getStructEq({
-  x: eq.eqNumber,
-  y: eq.eqNumber,
-});
+const eqPoint: Eq<Point> = {
```

Comment:
We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

Reply:
Fixed it.